thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java

```diff
@@ -211,12 +244,15 @@ public void buildReport(long startTime, long endTime, Long groupId, String group
       templateData.put("trueAnomalyCount", precisionRecallEvaluator.getTrueAnomalies());
       templateData.put("falseAlertCount", precisionRecallEvaluator.getFalseAlarm());
       templateData.put("newTrendCount", precisionRecallEvaluator.getTrueAnomalyNewTrend());
-      templateData.put("anomalyDetails", anomalyReportDTOList);
-      templateData.put("alertConfigName", alertConfigName);
+      templateData.put("anomalyDetails", anomalyReports);
+      templateData.put("groupedAnomalyDetails", groupedAnomalyReports.asMap());
+      templateData.put("emailSubjectName", emailSubjectName);
+      templateData.put("alertConfigName", alertConfig.getName()); // NOTE: breaks backwards compatibility
```

Comment:
note this change. email subject name used to overload alertConfig.getName() in some circumstances. they are now separated.